src/lib/default-project/project-data.js

```diff
@@ -0,0 +1,150 @@
+import {defineMessages} from 'react-intl';
+
+const messages = defineMessages({
+    backdrop: {
+        defaultMessage: 'backdrop1',
```

Comment:
Seeing it written out like this makes me wonder why we don't use spaces in these names... They aren't variable names or anything. It's definitely too late to change it though :/